content/en/docs/tasks/administer-cluster/cluster-management.md

```diff
@@ -1,223 +0,0 @@
-then the node controller will terminate the pods that are bound to the unavailable node.  If there is a corresponding
-replica set (or replication controller), then a new copy of the pod will be started on a different node.  So, in the case where all
-pods are replicated, upgrades can be done without special coordination, assuming that not all nodes will go down at the same time.
-
-If you want more control over the upgrading process, you may use the following workflow:
-
-Use `kubectl drain` to gracefully terminate all pods on the node while marking the node as unschedulable:
-
-```shell
-kubectl drain $NODENAME
-```
-
-This keeps new pods from landing on the node while you are trying to get them off.
-
-For pods with a replica set, the pod will be replaced by a new pod which will be scheduled to a new node. Additionally, if the pod is part of a service, then clients will automatically be redirected to the new pod.
-
-For pods with no replica set, you need to bring up a new copy of the pod, and assuming it is not part of a service, redirect clients to it.
-
-Perform maintenance work on the node.
-
-Make the node schedulable again:
-
-```shell
-kubectl uncordon $NODENAME
-```
-
-If you deleted the node's VM instance and created a new one, then a new schedulable node resource will
-be created automatically (if you're using a cloud provider that supports
-node discovery; currently this is only Google Compute Engine, not including CoreOS on Google Compute Engine using kube-register).
-See [Node](/docs/concepts/architecture/nodes/) for more details.
-
-## Advanced Topics
-
-### Turn on or off an API version for your cluster
-
-Specific API versions can be turned on or off by passing `--runtime-config=api/<version>` flag while bringing up the API server. For example: to turn off v1 API, pass `--runtime-config=api/v1=false`.
-runtime-config also supports 2 special keys: api/all and api/legacy to control all and legacy APIs respectively.
-For example, for turning off all API versions except v1, pass `--runtime-config=api/all=false,api/v1=true`.
-For the purposes of these flags, _legacy_ APIs are those APIs which have been explicitly deprecated (e.g. `v1beta3`).
-
-### Switching your cluster's storage API version
-
-The objects that are stored to disk for a cluster's internal representation of the Kubernetes resources active in the cluster are written using a particular version of the API.
-When the supported API changes, these objects may need to be rewritten in the newer API.  Failure to do this will eventually result in resources that are no longer decodable or usable
-by the Kubernetes API server.
-
-### Switching your config files to a new API version
-
-You can use `kubectl convert` command to convert config files between different API versions.
-
-```shell
-kubectl convert -f pod.yaml --output-version v1
-```
-
-For more options, please refer to the usage of [kubectl convert](/docs/reference/generated/kubectl/kubectl-commands#convert) command.
```

Comment:
Where is this content removed?

Reply:
Do you mean moved to?
Since there is an open issue referring to this content, I suspect that it does not make sense to completely remove the content before creating modified (or new) content as a replacement.
@sftim, could you leave the file in place or propose a substitute file as a replacement?

Reply:
I've added 2 new tasks, based on that removed text:
- [Enable Or Disable A Kubernetes API](https://deploy-preview-24415--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/enable-disable-api/)
  - needs tech review; I think the original text was outdated.
- [Upgrade A Cluster](https://deploy-preview-24415--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/cluster-upgrade/)
  - feels like a useful task to document, right?
  - mostly signposting to the kubeadm docs
  - also has some advice about what to do post-upgrade

These aren't complete, but at least we're not losing what's there..